detectorSegmentations/include/detectorSegmentations/FCCSWGridModuleThetaMerged_k4geo.h

```diff
@@ -94,6 +95,20 @@ class FCCSWGridModuleThetaMerged_k4geo : public GridTheta_k4geo {
    */
   inline const std::string& fieldNameModule() const { return m_moduleID; }
 
+  /// Extract the layer index fom a cell ID.
+  int layer(const CellID& aCellID) const;
+
+  /// Determine the volume ID from the full cell ID by removing all local fields
+  virtual VolumeID volumeID(const CellID& cellID) const override;
+
+  /// Return true if this segmentation can have cells that span multiple
+  /// volumes.  That is, points from multiple distinct volumes may
+  /// be assigned to the same cell.
+  virtual bool cellsSpanVolumes() const override
```

Comment:
```suggestion
  virtual bool cellsSpanVolumes() const
```
Does it work to maintain compatibility with the current release? I believe it would compile but I'm not sure if the results will be correct. Otherwise we should add the version of DD4hep that we require in the top level `CMakeLists.txt`.

Reply:
Yeah, i think that would compile.  The results would be wrong... but maybe not more wrong than they already are.
That still feels like the the wrong way to go about it, though.

Reply:
It's Ok, then if compatibility can not be maintained please bump the requirement for DD4hep here: https://github.com/key4hep/k4geo/blob/main/CMakeLists.txt#L47

Reply:
For now, i went ahead and removed the override keywords.  (Need to remove all of them, not just on the problematic function, otherwise we may get warnings from clang.)
Will add them back at a later time.

Reply:
People should be aware then not to use it with the stable release for the reasons above, although I'm assuming for development they are probably using the nightlies anyway.